cookbooks/cdo-varnish/attributes/default.rb

```diff
@@ -26,6 +26,14 @@
     'csedweek.net'        => 'csedweek.org',
     'cseducationweek.com' => 'csedweek.org',
     'cseducationweek.org' => 'csedweek.org',
+    'ar.code.org'     => 'program.ar/la-hora-del-codigo/',
+    'br.code.org'     => 'programae.org.br/',
+    'italia.code.org' => 'programmailfuturo.it',
+    'ro.code.org'     => 'adfaber.org/HourofCode/',
+    'sg.code.org'     => 'code.org',
+    'tr.code.org'     => 'robincode.org',
+    'uk.code.org'     => 'code.org',
+    'za.code.org'     => 'codeforchange.co.za/',
```

Comment:
@wjordan Is this the right place to do such redirects?

If so, what's the best way to test that this works?  

And is it safe to include the removal of these sites from pegasus in this PR as well?

Reply:
I haven't touched or looked at any of this code since 2015 so I can't answer any of your questions confidently, but I'll give my best guesses.

- I think this is an OK place to do redirects
- Test manually after merging the attribute changes- this feature was never developed in a way that could be tested in an isolated environment. Note that a redirect for `ar.code.org` won't also provide a redirect for `staging-ar.code.org` here, so you'll need to include those if you want to test on staging, otherwise just test once it's pushed to production.

- Removing the subdomains from `CDO.partners` will remove them from the CloudFront distribution, which will cause those requests to fail unless you manually update the DNS entries first to send requests directly to the load balancer instead of through the CloudFront distribution. Each partner has the following manual DNS entries:

- `staging-za.code.org`: `ALIAS staging.code.org.`
- `test-za.code.org`: `ALIAS test.code.org.`
- `za.code.org`: `ALIAS code.org.`
- `localhost.za.code.org`: `ALIAS localhost.code.org.`

I'd suggest deleting all but the production one, which should have its alias updated to point to the `production-codeorg` load balancer.

Beyond that, I can't think of anything else that will definitely break from removing the partner sites in this PR, but I can't guarantee safety beyond whatever our tests currently cover.